README.md

```diff
@@ -657,12 +665,108 @@ Basic usage: `amclient.py <subcommand> [optional arguments]
+
+* get-ingest-status
+  * purpose: print the status of an ingest if it exists in an ingest workflow.
+  * positional arguments
+    * sip-uuid - identifier for the ingest in progress
+    * am_api_key - API key for Archivematica dashboard user
+  * optional arguments
+    * `--am-user-name <username>` - username for Archivematica dashboard user
+    * `--am-url <url>` - Archivematica URL (default: `http://127.0.0.1`)
+
+* get-processing-config
+  * purpose: print a processing configuration file given its name in
+  Archivematica.
+  * positional arguments
+    * am_api_key - API key for Archivematica dashboard user
+  * optional arguments
+    * processing-config
+    * `--am-user-name <username>` - username for Archivematica dashboard user
+    * `--am-url <url>` - Archivematica URL (default: `http://127.0.0.1`)
+
+* approve-transfer
+  * purpose: approve a transfer in the Archivematica pipeline with a given
+  UUID.
+  * positional arguments
+    * transfer-directory <path> - relative path within the Transfer Source
+    * am_api_key - API key for Archivematica dashboard user
+  * optional arguments
+    * `--transfer-type` - type of transfer being initiated (default:
+    `standard`)
+    * `--am-user-name <username>` - username for Archivematica dashboard user
+    * `--am-url <url>` - Archivematica URL (default: `http://127.0.0.1`)
+
+* reingest-aip
+  * purpose: initiate the reingest of an AIP from the storage service with a
+  given UUID.
+  * positional arguments
+    * pipeline-uuid - identifier for the pipeline to reingest on
+    * aip-uuid - identifier for the AIP to reingest
+    * ss_api_key - Storage Service API key
+  * optional arguments
+    * `--reingest-type` - reingest pathway to use (default: `full`)
+    * `--processing-config` - named processing configuration to reingest with
+    (default: `default`)
+    * `--ss-user-name <username>` - Storage Service username (default: `test`)
+    * `--ss-url <url>` - Storage Service URL (default: `http://127.0.0.1:8000`)
+
+* get-package-details
+  * purpose: retrieve details about a package in the storage service with a
+  given UUID.
+  * positional arguments
+    * package-uuid - identifier of the package to retrieve details about
+    * ss_api_key - Storage Service API key
+  * optional arguments
+    * `--ss-user-name <username>` - Storage Service username (default: `test`)
+    * `--ss-url <url>` - Storage Service URL (default: `http://127.0.0.1:8000`)
+
 In addition, these optional arguments are available for all subcommands:
 * `--help`, `--h` - show help message and exit
 * `--output-mode <mode>` - how to print output, JSON (default) or Python
 
 See notes above about finding the Archivematica and Storage Service API keys.
 
+Reingest
+--------
+
+The *transfers/reingest.py* script is a module and CLI that provides
+functionality for bulk reingest of compressed AIPs. Given a user formatted
+list of AIP UUIDs then it can also complete the bulk-reingest of any AIP
```

Comment:
Seems odd to use "bulk reingest" and then "bulk-reingest" right afterwards. Is the distinction semantically warranted?

I would remove the "then" and the "also" making it "Given a user formatted
list of AIP UUIDs, it can complete the bulk-reingest of any AIP listed." Why is this "also"? Isn't it just elaborating on the first sentence?